File: Acceso a Datos - Ruth/workspaceEclipse/Tarea2_Contexto/src/main/java/servlets/Persona.java
package servlets;

public class Persona {

	String nombre;
	String edad;
	String tlfn;
	
	public Persona(String nombre, String edad, String tlfn) {
		super();
		this.nombre = nombre;
		this.edad = edad;
		this.tlfn = tlfn;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getEdad() {
		return edad;
	}

	public void setEdad(String edad) {
		this.edad = edad;
	}

	public String getTlfn() {
		return tlfn;
	}

	public void setTlfn(String tlfn) {
		this.tlfn = tlfn;
	}

	@Override
	public String toString() {
		return "Persona [nombre=" + nombre + ", edad=" + edad + ", tlfn=" + tlfn + "]";
	}
	
	
	
}

File: Moviles - Victor/Ejercicios/Ejer02_ComponentesBasicosDatabinding/app/src/main/java/com/eirmarse/ejercicio2/Persona.kt
package com.eirmarse.ejercicio2

class Persona(var nombre:String,var soltero:Boolean=true, var deportes:List<String> = emptyList()
              , var edad:Int=0) {
    override fun toString(): String {
        return "Persona(nombre='$nombre', soltero=$soltero, deportes=$deportes, edad=$edad)"
    }
}